### ISSUE TITLE
Decide on templating framework(s)

### ISSUE THREAD

**Issue by superbunker** [PITRI-9-Yeti-Crab/Scratch-Project#3]
- [ ] evaluate MaterialUI 
- [ ] evaluate Bootstrap 
- [ ] evaluate Tailwind

**Comment by nAndrewT** [comment]
Material UI and Bootstrap do indeed look to be the most useful for us. There are a few interesting others, but they have some drawbacks. Blueprint.js looks awesome, but I think it's way overkill. Ant Design looks cool, but it sounds like there's some stuff going on under the hood that make it a bit of a pain to work with, especially with testing.

MUI and Bootstrap are "classics" and there seems to be more info about them online than many other libraries. 

No matter what we use, I'm surprised by some of the complexity we will be dealing with. As Jeff said the other night, MUI is not plug and play. But that's ok!

**Comment by superbunker** [comment]
I'm leaning towards Bootstrap for simplicity's sake—we can just get responsive tables working for the lists and go from there. 

This could maybe be a starting point template?
https://getbootstrap.com/docs/4.0/examples/offcanvas/?#

**Comment by nAndrewT** [comment]
I'm playing around right now with react-bootstrap. We should definitely use it. The components are slick and easy to use. I'm not sure how to customize things yet, but we'll figure it out. I'm not seeing any templates in react-bootstrap. Is that just a straight Bootstrap thing? Either way, I have installed react-bootstrap and bootstrap in my branch. Bootstrap is v5.2.3 if that makes any difference to you. I read they changed some things (no more jQuery, for example).

EDIT 

Later on, I got into MUI as well. It's also very cool and has maybe more advanced components. I've gotten it to work in my branch.

As for templates, I'm unsure about them. A lot of the free Bootstrap templates I found use jQuery. I'm not interested in that. The MUI templates might be more feasible and easier to understand, but still require some grappling with. Both Bootstrap and MUI have a lot of value. Looking forward to digging in.

I won't push anything to Dev tonight. Let's wait until we have some solid ideas before that. At this point, the push might muddy things up just a bit.
